### ISSUE TITLE
Mdlink is not parsing image correctly

### ISSUE THREAD

**Issue by glitchlover** [reuseman/flashcards-obsidian#153]
The issue first arrived here

https://github.com/reuseman/flashcards-obsidian/issues/150#issue-1418692695

after researching, I found extra "25" number after "%"  in anki flashcard's souce code
```src="Pasted%2520image%252020221024224835.png"```
<img width="367" alt="image" src="https://user-images.githubusercontent.com/59877631/197582949-6d97f95c-97fe-402f-ace9-39b652b74f89.png">

if I remove the extra 25 it gets correct
```Pasted%20image%2020221024224835.png```
<img width="354" alt="image" src="https://user-images.githubusercontent.com/59877631/197583085-9fa4d867-b403-478a-8710-de98a27862ab.png">

**Comment by reuseman** [comment]
So if you paste an image in Obsidian there is an extra "25"? Or is it a problem of the plugin?

**Comment by glitchlover** [comment]
@reuseman no, it has nothing to do with obsidian. I am seeing this problem after parsing card to anki. Moreover, I am seeing more cases of broken image today. It will take time to organize the cases in where these things happen. I will update data soon.

**Comment by wawa-dan** [comment]
Hello @glitchlover, 
what is mdlink ? 
I reproduce the issue but without same "25" added.
Instead in my side, the slash is replace in anki source by a %2F, but when I go throw the anki media folder : C:\Users\xxx\AppData\Roaming\Anki2\Utilisateur 1\collection.media\   the media exist without the folder name and slash. 

Probably different issue between OS, used characters.

**Comment by wawa-dan** [comment]
Nb : can't we continue directly in the issue 150 and stop the 153? quite confusing to have 2

**Comment by reuseman** [comment]
This v1 encoding bug is fixed in Flashcards 2.0.0. URL-escaped Markdown filenames are decoded once for resolution and replaced with a content-hashed Anki media filename, so spaces no longer become `%2520`. See the [2.0.0 release](https://github.com/reuseman/flashcards-obsidian/releases/tag/2.0.0).
